exercises/difference-of-squares/difference_of_squares.py

```diff
@@ -1,10 +1,10 @@
-def square_of_sum():
+def square_of_sum(number_of_numbers):
```

Comment:
This doesn't sound natural. Maybe a better option can be found?

Reply:
Only for this function or for all the functions? Perhaps just `N` would suffice now that I think about it, given the reference in the README.md?

Reply:
for all of them of course 😄 
I don't like `N`  even it is stated in `README.md` that way, it develops a bad habit of naming your variables\parameter names to ambiguous things and your code suffers from it. Do you see my point?

A good Python style guide from Google - https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming